compiler/src/iree/compiler/Codegen/Common/LoweringConfigInterpreter.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <utility>
+#include "iree/compiler/Codegen/Common/Passes.h"
+#include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenDialect.h"
+#include "mlir/Dialect/Transform/IR/TransformDialect.h"
+#include "mlir/Dialect/Transform/IR/TransformOps.h"
+#include "mlir/Dialect/Transform/Transforms/TransformInterpreterUtils.h"
+#include "mlir/IR/SymbolTable.h"
+#include "mlir/Pass/Pass.h"
+
+using namespace mlir;
+
+namespace mlir::iree_compiler {
+
+#define GEN_PASS_DEF_LOWERINGCONFIGINTERPRETERPASS
+#include "iree/compiler/Codegen/Common/Passes.h.inc"
+
+namespace {
+
+class LoweringConfigInterpreterPass final
+    : public impl::LoweringConfigInterpreterPassBase<
+          LoweringConfigInterpreterPass> {
+public:
+  using impl::LoweringConfigInterpreterPassBase<
+      LoweringConfigInterpreterPass>::LoweringConfigInterpreterPassBase;
```

Comment:
```suggestion
  using Base::Base;
```
I think this should work

Reply:
Good to know, thanks!